coveralls/api.py

```diff
@@ -108,15 +108,23 @@ def load_config_from_github(self):
         # the same -- forceibly using Github's flow seems to be more stable
         self.config['repo_token'] = os.environ.get('GITHUB_TOKEN')
 
+        # TODO: Clarify default service.
+        # From experiments, it seems that service should always be set to
+        # 'github'. However, that is not yet confirmed - for discussion, see
+        # https://github.com/lemurheavy/coveralls-public/issues/1710.
+        # For compatibility, the service continues to be set to 'github-actions'
+        # for now. Users can specify the service name manually to override
+        # this.
+        service = 'github-actions'
```

Comment:
Actually service should be `github` to integrate with Github properly.

```suggestion
        service = 'github'
```

Reply:
I agree, I just did not want to change this without being sure. I think that the discussion in https://github.com/lemurheavy/coveralls-public/issues/1710 also argues for using "github".